src/N3Parser.js

```diff
@@ -1010,8 +1010,8 @@ export default class N3Parser {
 
   // ## Public methods
 
-  // ### `parse` parses the N3 input and emits each parsed quad through the callback
-  parse(input, quadCallback, prefixCallback) {
+  // ### `parse` parses the N3 input and emits each parsed quad through the callback. Optionally you can set a prefixCallback and a commentCallback
+  parse(input, quadCallback, prefixCallback, commentCallback) {
```

Comment:
We're getting too many arguments here; let's support:

- `parse(input, onQuad)`
- `parse(input, onQuad, onPrefix)`
- `parse(input, { onQuad, onPrefix, onComment })`

the first two of which are for legacy compatibility.

Reply:
Ok, will amend my PR accordingly